src/main/webapp/files/api.json

```diff
@@ -1,5 +1,5 @@
 {
-    "basePath": "http://localhost:8080/",
+    "basePath": "http://localhost:8080/media",
```

Comment:
Why does this keep changing? Also, do you need basePath? Seems like a hardcoding.

Reply:
Please answer the question above, if basePath is not needed, it can probably be removed.

Reply:
That file is dynamically generated at runtime (see [here](https://github.com/cormorack/cava-media/blob/afec2a901ecf955f800af2966141e0a44eee8e0f/grails-app/controllers/cavamedia/MediaController.groovy#L248)).  It shouldn't be under version control which is why I've added it to .gitignore.